docs/Makefile

```
@@ -1,31 +1,21 @@
 .PHONY: build-containerized
 build-containerized:
 	$(RUNTIME) volume create --ignore hypershift-docs-site
 	$(RUNTIME) run --rm -it \
-		-v ${PWD}:/home/docs/hypershift:Z \
-		-v hypershift-docs-site:/home/docs/hypershift/site \
+		-v ${PWD}:/docs:Z \
+		-v hypershift-docs-site:/docs/site \
 		$(IMG) build --strict
 
 .PHONY: serve-containerized
 serve-containerized:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)